newrelic-php-agent/values.yaml

```diff
@@ -1,11 +1,48 @@
-# Default values for simple-echo.
+# Default values for newrelic-php-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-image:
-  repository: chatwork/newrelic-php-agent
-  tag: latest
-  pullPolicy: IfNotPresent
-  command: []
-
-nameOverride: ""
-fullnameOverride: ""
\ No newline at end of file
+newrelic:
+  image:
+    repository: chatwork/newrelic-php-agent
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+    command: []
+
+  address: /tmp/.newrelic.sock
+
+  port: 31339
```

Comment:
@cw-niinuma 
default uses `/tmp/.newrelic.sock`.
Isn't `port` empty by default in `values.yaml`?

Reply:
@cw-niinuma 
https://github.com/chatwork/charts/pull/118/files#diff-5def190c92090a6bbbdc0d41d878c4beR28
`nil`?